src/engine/input_name.asm

```diff
@@ -445,32 +469,47 @@ NamingScreen_CheckButtonState:
 	ret nz
 	ld a, [wVisibleCursorTile]
 	bit 4, [hl]
-	jr z, Func_1aa07.asm_6a0a
+	jr z, DrawCursor_PlayerNamingScreen
+;	fallthrough
 
-Func_1aa07:
+DrawInvisibleCursor_PlayerNamingScreen:
```

Comment:
Rather name it PlayerNamingScreen_DrawInvisibleCursor

Reply:
If this isn't used outside the scope of this routine, why use it as a global label?